README.md

```diff
@@ -16,3 +18,6 @@ After the import you should see a return value of 1011 when running `Item.count`
 
 All users have a password of `password` if you want to login and explore.
 
+## Testing
+
+This application is tested using RSpec. Run `rspec` from the command line to run the test suite.
```

Comment:
Do we also want them to run `rake db:test:prepare` ?

Reply:
Also, it might be nice to add a source here. Maybe instead of "RSpec", we could add something like "[Rspec-Rails](https://github.com/rspec/rspec-rails)"?

Reply:
On the initial opening of the application, we didn't need to run this command, but I think it's a great addition for when we add additional migrations.

I'll change it now